cmd/version.go

```diff
@@ -0,0 +1,34 @@
+package cmd
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	// Version Contains the version number for a particular Resumic build
+	version = "dev"
+	// GitCommit contains the commit that this build is based on
+	gitCommit = "none"
+)
+
+const packageName string = "Resumic"
+
+func versionFunc(cmd *cobra.Command, args []string) error {
+	fmt.Println(packageName + "-" + version)
+	fmt.Println(runtime.GOOS + ", " + runtime.Version() + ", " + gitCommit)
+	return nil
```

Comment:
What's the reasoning behind having a possible error return value, but not having an error case within the function? Is this something that needs to be done due to cobra or preparation for future failure cases?

Reply:
Yeah, it's a Cobra requirement.

Reply:
Thanks for clarifying. Then it's "nothing to see, keep walking" :P